catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2482,73 +2418,45 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
+                if (source != null) {
+                    LOGGER.debug("Adding federated site to federated query: {}",
+                            source.getId());
+                }
 
-                    if (source != null) {
-                        LOGGER.debug("Retrieving product from remote source {}", source.getId());
-                        ResourceRetriever retriever = new RemoteResourceRetriever(source,
-                                responseURI,
-                                requestProperties);
-                        try {
-                            resourceResponse =
-                                    frameworkProperties.getReliableResourceDownloadManager()
-                                            .download(resourceRequest, metacard, retriever);
-                        } catch (DownloadException e) {
-                            LOGGER.info("Unable to download resource", e);
-                        }
-                    } else {
-                        LOGGER.warn("Could not find federatedSource: {}", resourceSourceName);
-                    }
-                } else {
-                    LOGGER.debug("Retrieving product from local source {}", resourceSourceName);
-                    ResourceRetriever retriever =
-                            new LocalResourceRetriever(frameworkProperties.getResourceReaders(),
-                                    responseURI,
-                                    requestProperties);
+                if (source != null) {
+                    LOGGER.debug("Retrieving product from remote source {}", source.getId());
+                    ResourceRetriever retriever = new RemoteResourceRetriever(source,
+                            responseURI,
+                            requestProperties);
                     try {
-                        resourceResponse = frameworkProperties.getReliableResourceDownloadManager()
-                                .download(resourceRequest, metacard, retriever);
+                        resourceResponse =
+                                frameworkProperties.getReliableResourceDownloadManager()
+                                        .download(resourceRequest, metacard, retriever);
                     } catch (DownloadException e) {
                         LOGGER.info("Unable to download resource", e);
                     }
+                } else {
+                    LOGGER.warn("Could not find federatedSource: {}", resourceSourceName);
+                }
+            } else {
+                LOGGER.debug("Retrieving product from local source {}", resourceSourceName);
+                ResourceRetriever retriever =
+                        new LocalResourceRetriever(frameworkProperties.getResourceReaders(),
+                                responseURI,
+                                requestProperties);
+                try {
+                    resourceResponse = frameworkProperties.getReliableResourceDownloadManager()
```

Comment:
Seen this a couple of times too. Any benefit to pulling the manager into a local/state variable? Or would that be a headache since we're in the longest file ever?